src/MsgProcessing.chpl

```diff
@@ -98,9 +98,12 @@ module MsgProcessing
         mpLogger.debug(getModuleName(),getRoutineName(),getLineNumber(), 
                                      "cmd: %s array: %s".format(cmd,st.attrib(name)));
         // delete entry from symbol table
-        st.deleteEntry(name);
-
-        repMsg =  "deleted %s".format(name);      
+        if st.deleteEntry(name) {
+            repMsg = "deleted %s".format(name);
+        }
+        else {
+            repMsg = "failed to delete registered symbol %s".format(name);
```

Comment:
Just a nit...I dunno about the repMsg -> the delete request doesn't fail in the case; it's not executed per the registration biz rules.

Reply:
Ah, that makes sense!